nl_NL/LC_MESSAGES/login.po

```diff
@@ -253,11 +253,11 @@ msgid "EMAIL_VERIFICATION_ADDRESS_REMOVED"
 msgstr "Sorry daarvoor! We hebben gemarkeerd dat je e-mailadres niet klopt, dus je hoort vanaf nu geen e-mails meer van Sudomemo te ontvangen."
 
 msgid "THANKS_FOR_LOGIN_PROMPT_INTRO"
-msgstr "Welcome to Sudomemo! Now, let's write a little message to say who you are! (This will be posted as a Flipnote in our Introductions channel.)"
+msgstr "Welkom op Sudomemo! Nu, Laten we een klein berichtje achter om te zeggen wie je bent! (Dit zal worden gepost als een flipnote in de "introduction" kanaal.)"
 
 msgid "PLEASE_DISCONNECT_AND_RECONNECT"
-msgstr "An error occured. To try again, disconnect and then reconnect to Sudomemo."
+msgstr "Een probleem is opgetreden Verbindt opnieuw met Sudomemo."
 
 # A URL will be shown at the end of this.
 msgid "LEGACY_PROXY_WARNING"
-msgstr "It looks like you may still be using the legacy ds.sudomemo.net proxy, which we have ended support for. As such, you should turn off the proxy in your System Settings, or you might have problems using Sudomemo. For more information, visit:"
+msgstr "Het lijkt erop dat je nog steeds onze oude proxy (ds.sudomemo.net) gebruikt. We raden je aan om de proxy te verwijderen en gewoon te verbinden met de dns. Voor meer informatie kijk op: %d"
```

Comment:
"DNS" should be capitalized. Running this through Google Translate it appears that it's saying that we recommend to use the DNS only. We don't recommend it, using the DNS only is required in order to access the service. The wording is slightly weird in the original English, however it should be more assertive.